openpype/hosts/maya/plugins/load/actions.py

```diff
@@ -29,18 +30,27 @@ def load(self, context, name, namespace, data):
         version = context['version']
         version_data = version.get("data", {})
 
-        start = version_data.get("frameStart", None)
-        end = version_data.get("frameEnd", None)
+        frame_start = version_data.get("frameStart", None)
+        frame_end = version_data.get("frameEnd", None)
 
-        if start is None or end is None:
-            print("Skipping setting frame range because start or "
-                  "end frame data is missing..")
+        if frame_start is None or frame_end is None:
+            print((
+                "Skipping setting frame range because start or"
+                " end frame data is missing.."
+            ))
             return
 
-        cmds.playbackOptions(minTime=start,
-                             maxTime=end,
-                             animationStartTime=start,
-                             animationEndTime=end)
+        handle_start = version_data.get("handleStart")
+        handle_end = version_data.get("handleStart")
+        frame_info = get_frame_info(
```

Comment:
The version data in the database/backend will always be inclusive. No? The database need to either choose one of the two formats or additionally should publish along whether it's inclusive or not. But `get_frame_info` applies the current settings to it and thus results in different frame ranges to be set. However once the project settings change that should NOT influence already published content.

Again. The handles being inclusive or not is solely a visual thing:
1. On create instances.
2. In the tools that display frame info.

Reply:
> And created instance don't have frame info in metadata, as far as I know.

They actually have:
![image](https://user-images.githubusercontent.com/33513211/156536237-38d13499-5ac0-4f01-ab68-4895926941d2.png)

Reply:
That ☝️ - and I'd still prefer changing the admin setting doesn't influence my existing scenes so I'd recommend adding a "how to interpret handles" attribute onto the instances.

Reply:
And where they come from? It's taken from Maya timeline?

Reply:
> And where they come from? It's taken from Maya timeline?

yes, these all taken from timeline set in Maya that is initialized from asset data (but can be changed during the work ofc)

Reply:
So they're always stored in scene as additive handles, no matter what is set on project.

Reply:
So even for the studios using the subtractive handles the settings on instances in the scene are considered additive?

I am not sure I understand. Where then does it actually influence the experience for those users?

Reply:
I would say that only place where it is affected are asset documents (project management).

I had in mind that all other places are defined by dcc itself (e.g. maya timeline probably can't show subtractive handles, or?) or are "metadata" driven so "User"/"Artist" should not see and care about it.

But as I can see I am wrong because these are "metadata" but can be modified by an artist. I wonder how he know that he should set them (e.g. `handleStart` or `handleEnd`)?

The question is if this should be affected by project settings too? If yes then are there other places where frame ranges are defined in metadata which are meant to be changed by an artist?